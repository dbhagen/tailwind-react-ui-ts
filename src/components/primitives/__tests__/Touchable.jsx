import React from 'react'
import { shallow } from 'enzyme'

import { Touchable } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign({}, testProps)

  const wrapper = shallow(<Touchable {...props}>Hello World</Touchable>)

  return {
    props,
    wrapper,
  }
}

describe('Touchable', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
