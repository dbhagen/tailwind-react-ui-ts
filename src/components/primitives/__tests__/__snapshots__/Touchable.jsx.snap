// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Touchable renders matching snapshot 1`] = `
<WithTheme(Base)
  cursor="pointer"
  disabled={false}
  focusable={true}
  is="button"
  select="none"
>
  Hello World
</WithTheme(Base)>
`;
